python/llm/src/ipex_llm/utils/utils.py

```diff
@@ -17,7 +17,14 @@
 import sys
 import pathlib
 from ipex_llm.utils.common import invalidInputError, invalidOperationError
+import transformers
 
+trans_version = transformers.__version__
```

Comment:
You can move this to `__init__.py`, and import lib by `from ipex_llm.utils import BenchmarkWrapper`